inst/rmarkdown/templates/gigascience_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,548 @@
+    note: Contributed equally
+
+# affiliation: list of {id,name,address?}
+#   - authors' institutional affiliations
+affiliation:
+  - id: 1
+    name: Some Institute of Technology
+    address: Some Department, Street, City, State, Zip
+  - id: 2
+    name: Another University
+  - id: 3
+    name: British Academy of Film and Television Arts
+    address: 195 Piccadilly, London, England, UK W1J 9LN
+
+# runningauthor: author's name in running title
+runningauthor: Anonymous et al.
+
+# papercat (paper category) - one of:
+#   - Commentary
+#     - http://academic.oup.com/gigascience/pages/commentary
+#   - Data Note
+#     - http://academic.oup.com/gigascience/pages/data_note
+#   - Research
+#     - http://academic.oup.com/gigascience/pages/research
+#   - Review
+#     - http://academic.oup.com/gigascience/pages/review
+#   - Technical Note
+#     - http://academic.oup.com/gigascience/pages/technical_note
+papercat: Technical Note
+
+# abstract: abstract structured according to instructions for paper category
+abstract: |
+  The Abstract (250 words maximum) should be structured according to the instructions for the corresponding 'papercat' (paper category, see https://academic.oup.com/gigascience/pages/instructions_to_authors).
+
+# keywords: keywords presented as a semicolon-separated list below the abstract  
+keywords:
+  - Keyword 1
+  - keyword 2
+  - keyword 3 (Three to ten keywords representing the main content of the article)
+
+# keypoints: key points presented as a bullet list in a box at the top of the second page
+keypoints:
+  - This is the first point
+  - This is the second point
+  - One last point.
+
+# highlighting-macros: list of {latex_command+}
+#   - This may be for syntax-highlighting source-code, but I'm not sure
+#   - Value is a list; each list-member is a TeX snippet
+#   - This is included just before 'header-includes'
+
+# header-includes: list of {latex_command+}
+#   - Value is a list; each list-member is a TeX snippet
+#   - You can use 'header-includes' to set or override some TeX settings, e.g.,
+#   - This is included just before 'preamble'
+#   - concerning specific keys:
+#     - journal: Only "gigascience" and "general" are implemented now.
+#     - usepackage: use packages that are not declared in the TeX template used by pandoc
+#     - bibliographystyle: apples to the natbib TeX package
+#       - see http://ctan.math.washington.edu/tex-archive/macros/latex/contrib/natbib/natbib.pdf
+header-includes:
+  - \journal{gigascience}
+  - \usepackage{natbib}
+#   # the next line is part of the document-class, oup-contemporary
+#   - \bibliographystyle{vancouver-authoryear}
+
+# preamble: Tex literal (newlines will be stripped out)  
+#   - Value is a TeX snippet that will be collapsed to a single line
+# preamble: |
+#  \setcounter{errorcontextlines}{4}
+#  \setcounter{secnumdepth}{2}
+
+# document-class - the TeX document class - for GigaScience, muct be oup-contemporary
+document-class: oup-contemporary
```

Comment:
Pandoc's convention is `documentclass`: http://pandoc.org/MANUAL.html#variables-for-latex

Reply:
You need to update the LaTeX template accordingly (using `documentclass` instead of `document-class`): https://github.com/eschen42/rticles/blob/bcbf202bafa263322df63526ad291a97d80420e0/inst/rmarkdown/templates/gigascience_article/resources/template.tex#L6